addons/mail/static/src/models/composer/composer.js

```diff
@@ -613,6 +625,22 @@ function factory(dependencies) {
             compute: '_computeMentionedPartners',
             dependencies: ['textInputContent'],
         }),
+        /**
+         * List of partners that are recipients of message being composed with
+         * this composer.
+         * This is the merge between mentionned partners and suggested
+         * recipients. It hold the people who will recive a notification.
+         */
+        recipients: many2many('mail.partner', {
+            compute: '_computeRecipients',
+            dependencies: [
+                'thread',
+                'threadSuggestedRecipients',
+            ]
```

Comment:
`,` at the end

missing dependencies `mentionedPartners`, `suggestedRecipientInfoList.checked`, `suggestedRecipientInfoList.partner`